src/popup/components/Modals/MultisigVaultCreate.vue

```diff
@@ -20,6 +20,7 @@
       <div
         v-for="(signer, index) in signers"
         :key="`signer${index}`"
+        :data-cy="`signer-address-${index}`"
```

Comment:
I strongly recommend to not use dynamic test IDs.

Reply:
I changed it to `:data-signer-idx` to make it more clear